docs/source/cookbook/2_basic_auth.rst

```diff
@@ -39,37 +125,32 @@ To generate certificates signed by a private CA
     # Generate server private key
     openssl genpkey -algorithm RSA -out certs/server.key
 
-    # Generate server CSR with SAN
+    # Generate server CSR
     openssl req -new -key certs/server.key -out certs/server.csr -subj "/CN=etcd-server"
 
     # Generate server certificate signed by the CA with IP address subject alternative name
-    openssl x509 -req -days 365 -in certs/server.csr -CA certs/ca.crt -CAkey certs/ca.key -CAcreateserial -out certs/server.crt -extfile <(echo "subjectAltName=IP:127.0.0.1,IP:${HOST_IP}")
+    openssl x509 -req -days 365 -in certs/server.csr -CA certs/ca.crt -CAkey certs/ca.key -CAcreateserial -out certs/server.crt -extfile <(echo "subjectAltName=IP:${HOST_IP}")
 
     # Generate client private key
     openssl genpkey -algorithm RSA -out certs/client.key
 
-    # Generate client CSR with SAN
+    # Generate client CSR
     openssl req -new -key certs/client.key -out certs/client.csr -subj "/CN=etcd-client"
 
     # Generate client certificate signed by the CA
     openssl x509 -req -days 365 -in certs/client.csr -CA certs/ca.crt -CAkey certs/ca.key -CAcreateserial -out certs/client.crt
 
-    chmod 644 certs/ca.key certs/server.key certs/client.key
-
-``certs/*.crt`` is a file with a certificate in PEM format and ``certs/*.key`` is a file with a key in PEM format where ``*`` is a replacement for `ca`, `server` and `client`.
-
 Launching etcd
```

Comment:
Change to Etcd (capital letter). Check other places too.

Reply:
I used the same writing as in the official documentation - `etcd`. See https://etcd.io/